vhdbuilder/packer/install-dependencies.sh

```diff
@@ -124,6 +124,9 @@ if [[ $OS == $MARINER_OS_NAME ]]; then
     disableDNFAutomatic
     enableCheckRestart
     activateNfConntrack
+    if [[ "${ENABLE_CGROUPV2,,}" == "true" ]]; then
+      enableCgroupV2forAzureLinux
```

Comment:
let's do this in pre-install to boot into v2 for install-deps -- this will get us correct telemetry set up in https://github.com/Azure/AgentBaker/pull/3406

Reply:
Fixed